filter_plugins/padded_yaml.py

```diff
@@ -0,0 +1,44 @@
+#
+# This filter allows you to expand vars into 
+#   multi line yaml snippets 
+#   the first line is positioned with in the template
+#   the following lines are padded to level * indent spaces
+#  
+# author Lutz Lange <llange@redhat.com>
+# 
+# example filter " ... | to_padded_yaml( level=2 )
+# ToDo : - test with more complex structures. 
+#        - test indentitation feature ( level=2, indent=4 )
+#
+
+import yaml
+from ansible.utils.unicode import to_unicode
+
+def to_padded_yaml(*a, **kw):
```

Comment:
In general we have been using named parameters for filters, which is the same convention as upstream ansible.

Also, I think it might be better to apply the padding to all lines and adjust the templates accordingly.

overally, maybe something more like:
```
def to_padded_yaml(data, level=0, indent=2, **kw):
    try:
        transformed = yaml.safe_dump(data, indent=indent, allow_unicode=True, default_flow_style=False, **kw)
        padded = "\n".join([ " " * level * indent + line for line in transformed.splitlines()])
        return to_unicode(padded)
    except Exception as e:
        raise errors.AnsibleFilterError('Failed to convert: %s' % e)
```